src/tools/tidy/Cargo.toml

```diff
@@ -2,6 +2,3 @@
 name = "tidy"
 version = "0.1.0"
 authors = ["Alex Crichton <alex@alexcrichton.com>"]
-
-[dependencies]
-regex = "0.2"
```

Comment:
Looks like a Cargo.lock file here needs to be updated presumably because of this change.